core/frontend/services/rss/generate-feed.js

```diff
@@ -4,11 +4,8 @@ const cheerio = require('cheerio');
 const RSS = require('rss');
 const urlUtils = require('../../../shared/url-utils');
 const urlService = require('../url');
-let generateFeed;
```

Comment:
This pattern of declaring all internal module methods is outdated and doesn't seem to serve any specific purpose other than having visibility into which methods are present in the module. This kind of visibility is easy to have within modern IDEs, so decided to break this pattern in aim to fix "no-shadow" linting warning.